### PR TITLE
feat: added focus styles for interactive elements

### PR DESCRIPTION
**Pull request by Kyzyl-ool**

### CODE REVIEW

File: src/blocks/ContentLayout/ContentLayout.scss [hunk 1]
```diff
@@ -60,6 +60,12 @@ $block: '.#{$ns}content-layout-block';
         padding: $indentXL;
     }
 
+    &_theme {
+        &_dark {
+            --g-color-line-focus: var(--g-color-text-light-primary);
```
Comment (yuberdysheva) [inline src/blocks/ContentLayout/ContentLayout.scss:65]: What color is default for `--g-color-line-focus` in uikit and why can't we use ut?

File: src/blocks/Slider/Slider.tsx [hunk 2]
```diff
@@ -251,7 +251,7 @@ export const SliderBlock = (props: WithChildren<SliderProps>) => {
         return (
             // To have this key differ from keys used in renderDot function, added `-accessible-bar` part
             <Fragment key={`${index}-accessible-bar`}>
-                {slidesCountByBreakpoint > 1 && (
+                {slidesCountByBreakpoint > 0 && (
```
Comment (yuberdysheva) [inline src/blocks/Slider/Slider.tsx:254]: why did you change the value? Could it break something?

File: src/blocks/Slider/Slider.tsx [hunk 3]
```diff
@@ -276,19 +277,26 @@ export const SliderBlock = (props: WithChildren<SliderProps>) => {
         } else {
             currentSlideNumber = index + 1;
         }
+        return currentSlideNumber;
+    };
+    const isVisibleSlide = (index: number) => {
+        const currentIndexDiff = index - currentIndex;
 
+        return (
+            slidesCountByBreakpoint > 0 &&
```
Comment (yuberdysheva) [inline src/blocks/Slider/Slider.tsx:286]: Something wrong with a slider. I can tab trough cards and then if I click to the arrow it doesn't work right
https://disk.yandex.ru/i/7dRgmxUNMis7iA

File: src/components/ReactPlayer/ReactPlayer.tsx [hunk 4]
```diff
@@ -391,6 +391,7 @@ export const ReactPlayerBlock = React.forwardRef<ReactPlayerBlockHandler, ReactP
                             onProgress={onProgress}
                             onEnded={onEnded}
                             aria-label={ariaLabel}
+                            previewTabIndex={-1}
```
Comment (yuberdysheva) [inline src/components/ReactPlayer/ReactPlayer.tsx:394]: What does it mean?

File: src/sub-blocks/HubspotForm/HubspotForm.scss [hunk 5]
```diff
@@ -259,7 +264,15 @@ $block: '.#{$ns}hubspot-form';
         p {
             color: var(--g-color-text-primary);
             @include text-body-2();
-            margin: 0 0 20px;
+            margin: 20px 0;
```
Comment (yuberdysheva) [inline src/sub-blocks/HubspotForm/HubspotForm.scss:267]: Can you use variable instead of 20px?

**Review by yuberdysheva — CHANGES_REQUESTED** [review]

File: src/blocks/Slider/Slider.tsx (hunk 2 above)
Comment (Kyzyl-ool) [reply src/blocks/Slider/Slider.tsx:254]: I change it because when we have `slidesCountByBreakpoint === 1` the accessible bar didn't render and one was unable to find it vis screenreader

File: src/blocks/Slider/Slider.tsx (hunk 3 above)
Comment (Kyzyl-ool) [reply src/blocks/Slider/Slider.tsx:286]: Yep, this is an issue I think came from react-slick, I didn't manage to find a solution better than wrap each slide with `inert` attribute. But it may be risky, as slides not visible for the user, will be unavailable for focus

File: src/components/ReactPlayer/ReactPlayer.tsx (hunk 4 above)
Comment (Kyzyl-ool) [reply src/components/ReactPlayer/ReactPlayer.tsx:394]: It means that preview image will not be focusable with Tab. At the same time we have on preview image play button, which is also accessible via Tab. I thought it would be sufficient to have it focusable

File: src/sub-blocks/HubspotForm/HubspotForm.scss (hunk 5 above)
Comment (Kyzyl-ool) [reply src/sub-blocks/HubspotForm/HubspotForm.scss:267]: Yep

File: src/blocks/ContentLayout/ContentLayout.scss (hunk 1 above)
Comment (Kyzyl-ool) [reply src/blocks/ContentLayout/ContentLayout.scss:65]: Ok, let's use default focus color in page-constructor

**Comment by Kyzyl-ool** [comment]
Introduced focus variables:
```
--pc-color-line-focus-light
--pc-color-line-focus-dark
```

**Review by yuberdysheva — APPROVED** [review]

File: src/components/FullscreenImage/FullscreenImage.tsx [hunk 6]
```diff
@@ -44,19 +44,24 @@ const FullscreenImage = (props: FullscreenImageProps) => {
                     onClick={openModal}
                     style={imageStyle}
                 />
-                <div className={b('icon-wrapper', {visible: isMouseEnter})} onClick={openModal}>
+                <button
```
Comment (gorgeousvlad) [inline src/components/FullscreenImage/FullscreenImage.tsx:47]: did we reset button styles?

File: src/components/FullscreenMedia/FullscreenMedia.tsx [hunk 7]
```diff
@@ -28,43 +28,66 @@ const getMediaClass = (type: string) => b('modal-media', {type});
...
         return children();
     }
 
     return (
         <div className={b()}>
-            <div className={b('media-wrapper')} onClickCapture={openModal}>
+            <div
+                className={b('media-wrapper')}
+                onClickCapture={openModal}
+                onMouseOver={onShowFullScreenButton}
```
Comment (gorgeousvlad) [inline src/components/FullscreenMedia/FullscreenMedia.tsx:55]: :hover in css doesn't suite this case?

File: src/sub-blocks/PriceDetailed/PriceDetails/PriceDetails.tsx [hunk 8]
```diff
@@ -71,14 +71,14 @@ const PriceDetails = (props: PriceDetailsExtendProps) => {
 
     const getFoldableTitle = () => {
         return (
-            <div
+            <button
```
Comment (gorgeousvlad) [inline src/sub-blocks/PriceDetailed/PriceDetails/PriceDetails.tsx:74]: did we reset button styles?

**Review by gorgeousvlad — CHANGES_REQUESTED** [review]

File: src/components/FullscreenImage/FullscreenImage.tsx (hunk 6 above)
Comment (Kyzyl-ool) [reply src/components/FullscreenImage/FullscreenImage.tsx:47]: yep

File: src/sub-blocks/PriceDetailed/PriceDetails/PriceDetails.tsx (hunk 8 above)
Comment (Kyzyl-ool) [reply src/sub-blocks/PriceDetailed/PriceDetails/PriceDetails.tsx:74]: also yes

**Review by gorgeousvlad — APPROVED** [review]
